### ISSUE TITLE
Can't use @datorama/akita-ngdevtools in AOT

### ISSUE THREAD

**Issue by sgwatgit** [datorama/akita#60]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
When running `ng serve --aot` with `@datorama/akita-ngdevtools` configured like so...

```
@NgModule({
    imports: [
        ....
        AkitaNgDevtools.forRoot({ actionsBlacklist: ['@@INIT'] })]
})
export class AppModule {}
```

I receive the following error

```
ERROR in Error during template compile of 'AppModule'
  Function calls are not supported in decorators but 'factory' was called in 'AkitaNgDevtools'
    'AkitaNgDevtools' calls 'factory'.
```

## Expected behavior
The dev tools work in AOT.

## Minimal reproduction of the problem with instructions
Add the dev tools package and run `ng serve --aot`.

## What is the motivation / use case for changing the behavior?
AOT brings a lot of benefits in validating templates and code while developing features. It would be nice to run the Akita dev tools while building with AOT.

## Environment

<pre><code>
Angular CLI: 6.1.2
Node: 10.8.0
OS: win32 x64
Angular: 6.1.1
... animations, common, compiler, compiler-cli, core, forms ... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
----------------------------------------------------------- @angular-devkit/architect         0.7.2
@angular-devkit/build-angular     0.7.2
@angular-devkit/build-optimizer   0.7.2
@angular-devkit/build-webpack     0.7.2
@angular-devkit/core              0.7.2
@angular-devkit/schematics        0.7.2
@angular/cli                      6.1.2
@ngtools/webpack                  6.1.2
@schematics/angular               0.7.2
@schematics/update                0.7.2
rxjs                              6.2.2
typescript                        2.9.2
webpack                           4.9.2

Browser:
- [ X] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 10.8.0
- Platform:  Windows

Others:

</code></pre>

**Comment by NetanelBasal** [comment]
Thanks, I will take a look soon. In the meantime, you can use the old way:

```ts
export class AppComponent implements OnInit {
  constructor(private ngZone: NgZone) {
    if (!environment.production) {
      akitaDevtools(ngZone);
    }
  }
}
```

**Comment by sgwatgit** [comment]
By the way, I don't want to be one of those guys that just brings issues without gratitude.

Akita is awesome and your posts on Medium are awesome!

Thanks so much!

**Comment by NetanelBasal** [comment]
Thanks. Nice to hear.

**Comment by NetanelBasal** [comment]
Fixed in @datorama/akita-ngdevtools@1.0.2. Thanks for the report.
